### DISCUSSION TITLE
JSON parse problem

### DISCUSSION THREAD

**Discussion by cicwak** [Kong/insomnia#4269] [Help] [answered]
Hi, i have a problem with JSON parse in insomnia[
![image](https://user-images.githubusercontent.com/63852198/144708917-b0fa0fb0-98d7-4b67-a84c-2d4a8e562083.png)
](url)

**Comment by gh-andre** [comment, marked as answer]
Check  that `Content-Type` header in the response says `application/json; charset="utf-8"`.
